scripts/create_image.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+mkdir $1
+sudo debootstrap jammy $1  http://archive.ubuntu.com/ubuntu
+
+sudo arch-chroot $1 /bin/bash <<EOF
+
+export PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+rm /etc/resolv.conf
```

Comment:
no need for the rm, in the next line you used `>` instead of `>>` so that should already overwrite the file